composer.json

```diff
@@ -32,6 +36,12 @@
         }
     },
     "scripts": {
-        
+        "check-style": "vendor/bin/php-cs-fixer fix --dry-run --format=txt --verbose --diff --diff-format=udiff --config=.cs.php",
+        "fix-style": "vendor/bin/php-cs-fixer fix --config=.cs.php",
+        "test": "vendor/bin/phpunit",
+        "check-all": [
+            "@check-style",
+            "@test"
```

Comment:
Define multiple scripts. They're as follows:

- `check-style`: This will check coding style and don't fix incorrect style with `php-cs-fixer`.
- `fix-style`: This will be automatic to fix coding style with `php-cs-fixer`.
- `test`: This will do unit tests with PHPUnit.
- `check-all`: This is a collection scripts and it will execute `check-style` and `test` in order.